analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.SpinLock.cs

```diff
@@ -100,6 +100,50 @@ public void TryEnter_UseReturnValueToReleaseOnlyWhenNeeded()
             }
         }
 
+        // https://github.com/SonarSource/sonar-dotnet/issues/5522
```

Comment:
Maybe put the two reproducers into a separate class/namespace. Or just duplicate the comment on the other reproducer method too. This way it seems that only the first method is a reproducer for the issue.